internal/provider/routing_models.go

```diff
@@ -0,0 +1,32 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// routingListResourceModel maps the resource schema.
+type routingListResourceModel struct {
+	// ID is required for Framework acceptance testing
+	ID              types.String `tfsdk:"id"`
+	Name            types.String `tfsdk:"name"`
+	Description     types.String `tfsdk:"description"`
+	IsActive        types.Bool   `tfsdk:"is_active"`
+	ProxyUrl        types.String `tfsdk:"proxy_url"`
+	ResourceSetName types.String `tfsdk:"resource_set_name"`
+}
+
+// routingListResourceModel maps the resource schema.
+type routingResourceModel struct {
+	// ID is required for Framework acceptance testing
+	ID            types.String `tfsdk:"id"`
+	Name          types.String `tfsdk:"name"`
+	Description   types.String `tfsdk:"description"`
+	IsActive      types.Bool   `tfsdk:"is_active"`
+	ProxyUrl      types.String `tfsdk:"proxy_url"`
+	ResourceSetId types.String `tfsdk:"resource_set_id"`
```

Comment:
I'm not a fan of the ResourceSetId being part of the TF Schema - it's inherited from the Provider and feels like it should remain there

Reply:
I confirmed that X-Aembit-ResourceSet header does not interfere with the ResourceSetId field in the TF Schema.